client/my-sites/plan-features/index.jsx

```diff
@@ -90,6 +92,15 @@ export default connect( ( state, ownProps ) => {
 		rawPrice: getPlanRawPrice( state, planProductId /**, get from abtest **/ ),
 		planConstantObj: plansList[ ownProps.plan ],
 		billingTimeFrame: get( planObject, 'bill_period_label', '' ),
+		onUpgradeClick: ownProps.plan === PLAN_FREE ? noop : () => {
```

Comment:
Since this is a connected component, I think it's fine to have it here, though we may want to leave the option of overriding this by checking `ownProps.onUpgradeClick`